src/components/ModalDialog/ModalDialog.tsx

```diff
@@ -0,0 +1,67 @@
+import styled from "@emotion/styled";
+import { rgba } from "polished";
+import React, { FunctionComponent } from "react";
+import tw from "twin.macro";
+import { useLockBodyScroll } from "../../common/hook/useLockBodyScroll";
+import { vars } from "../../styles";
+
+interface ModalDialogProps {
+  className?: string;
+  show: boolean;
+  close: () => void;
+  children?: React.ReactNode;
+}
+
+export const ModalDialogUnStyled: FunctionComponent<ModalDialogProps> = ({
+  className,
+  show,
+  close,
+  children,
+}) => {
+  useLockBodyScroll();
+
+  return (
+    <>
+      {show && (
```

Comment:
Don't have to pass a show props in if it's handled by the parent

Reply:
Can remove the fragment while at it

Reply:
ok